src/kagglehub/kaggle_cache_resolver.py

```diff
@@ -39,7 +39,11 @@ def is_supported(self, *_) -> bool:
 
         return False
 
-    def __call__(self, h: ModelHandle, path: Optional[str] = None) -> str:
+    def __call__(self, h: ModelHandle, path: Optional[str] = None, *, force_download: Optional[bool] = False) -> str:
+        if force_download:
+            msg = "Invalid input: Cannot force download in a Kaggle notebook"
+            raise ValueError(msg)
```

Comment:
Actually, we should simply log a warning here.

The reason is that we want the same code to run regardless of whether users are in a Kaggle notebook or outside.